test/index.js

```diff
@@ -1,8 +1,10 @@
 'use strict'
```

Comment:
@bsiddiqui, I rewrote all the specs for the latest Lab and Code, using `await / async`. 

```
> bookshelf-history@1.0.0 test /Users/christophersun/Projects/homevest/bookshelf-history
> standard && lab --verbose --colors --assert @hapi/code --coverage

bookshelf-history
  ✔ 1) it runs with default arguments (22 ms and 5 assertions)
  ✔ 2) it detects patch operation (10 ms and 5 assertions)
  ✔ 3) it does not create tuples when history is disabled (2 ms and 2 assertions)
  ✔ 4) it disables backup when options.history is false (6 ms and 3 assertions)
backup()
  ✔ 5) it saves a manual history (6 ms and 2 assertions)
  ✔ 6) it accepts running transactions (5 ms and 2 assertions)
revert()
  ✔ 7) it reverts (15 ms and 5 assertions)
  ✔ 8) it accepts running transactions (11 ms and 5 assertions)
  ✔ 9) it uses latest sequence when none is provided (11 ms and 2 assertions)
  ✔ 10) it works with json datatypes on postgres (18 ms and 5 assertions)
history
  ✔ 11) works (8 ms and 4 assertions)
```